src/helpers/pairs.ts

```diff
@@ -40,6 +46,9 @@ export function getInterpolatorForPair(
 export function getInterpolatorForPair(
   pairs: CSSPairs
 ): InterpolatedResult<RGBA<number>, string>;
+export function getInterpolatorForPair(
+  pairs: CSSPairs
+): InterpolatedResult<string, string>;
```

Comment:
A new overload. In the case of values like "10px", the interpolator should return a string value as well with the same unit.